scripts/opentitan/autotop.py

```diff
@@ -510,7 +497,7 @@ def main():
     desc = sys.modules[__name__].__doc__.split('.', 1)[0].strip()
     argparser = ArgumentParser(description=f'{desc}.')
     try:
-        languages = list(QEMUAutoTop.languages)
+        outkinds = list(QEMUAutoTop.outkinds)
```

Comment:
Nit: already a `list`, so the conversion is redundant? Feels like it should also be an assert, given that we rely on it being non-empty.

```suggestion
        outkinds = QEMUAutoTop.outkinds
        assert len(outkinds) > 0
```

Reply:
> the conversion is redundant?

This is a quirk with the linter: it does not get that @classproperty returns an iterable, but for some reason does not catch it if we force an iteration. I need to dig into the stacked  Python decorations at some point to understand this issue.